engines/pyfunc-ensembler-service/app.Dockerfile

```diff
@@ -52,4 +52,4 @@ RUN if [ "${MLFLOW_ARTIFACT_STORAGE_TYPE}" = "gcs" ]; then  \
 ENV FOLDER_NAME=$FOLDER_NAME
 SHELL ["/bin/bash", "-c"]
 ENTRYPOINT . activate ${CONDA_ENV_NAME} && \
-  python -m pyfunc_ensembler_runner --mlflow_ensembler_dir ./${FOLDER_NAME} -l INFO
+  python -m pyfunc_ensembler_runner --mlflow_ensembler_dir ./${FOLDER_NAME} -l WARNING
```

Comment:
i prefer we remove this `-l WARNING` flag. since it isn't easy to update the value unless we update the whole container entrypoint
```bash
docker run --entrypoint ". activate ${CONDA_ENV_NAME} && python -m pyfunc_ensembler_runner --mlflow_ensembler_dir ./${FOLDER_NAME} -l DEBUG"
```